test/functional/apps/discover/group1/_discover_histogram.ts

```diff
@@ -65,13 +66,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     it('should modify the time range when the histogram is brushed', async function () {
       await PageObjects.common.navigateToApp('discover');
       await PageObjects.discover.waitUntilSearchingHasFinished();
-      await PageObjects.timePicker.setDefaultAbsoluteRange();
-      await PageObjects.discover.waitUntilSearchingHasFinished();
       // this is the number of renderings of the histogram needed when new data is fetched
       let renderingCountInc = 1;
       const prevRenderingCount = await elasticChart.getVisualizationRenderingCount();
-      await PageObjects.timePicker.setDefaultAbsoluteRange();
-      await PageObjects.discover.waitUntilSearchingHasFinished();
```

Comment:
there's an occasional error that setting the timerange here again leads to a double submit. then the test is failing:

```
debg Find.allByCssSelector('.echChart') with timeout=10000
[00:04:39]         â”‚ debg renderings before brushing - actual: 2 expected: 1
[00:04:39]         â”‚ debg Find.allByCssSelector('.echChart') with timeout=10000
[00:04:39]         â”‚ debg renderings before brushing - actual: 2 expected: 1
```
Since there's no reason the set the time range, just submitting the query should be sufficient for re-rendering the histogram